src/services/LbpProjectTokenPriceService.ts

```diff
@@ -0,0 +1,144 @@
+import { autoinject } from "aurelia-framework";
+import * as moment from "moment-timezone";
+import Moment = moment.Moment;
+
+@autoinject
+export class LbpProjectTokenPriceService {
+
+  maxSupply: number;
+  projectTokenInitialLbpAmount: number;
+  fundingTokenInitialLbpAmount: number;
+  startWeightProjectToken: number;
+  endWeightProjectToken: number;
+  startDate: string;
+  endDate: string;
+  roundedStartDate: Moment;
```

Comment:
rounder how?  can we do this rounding in the service rather than forcing the caller to do it?